Rutushree-TC0038-CartValueUpdateValidationTestcase

Rutu12Badwaik opened the pull request:
I create a methods for feature file steps which is assigned by your changes

sarangholey reviewed (commented):
Go ahead with the test step in step definition file

akashdktyagi commented on Batch8CucumberSeleniumAPI/src/test/java/runner/RunTempUI.java:
```diff
@@ -10,7 +10,7 @@
 		features="classpath:features/ui",
 		glue="stepdefs.ui",
-		tags="@TC0008",
+		tags="@TC0038",
```
No need to commit this file, you should only commit files which are relevant to the task you are working on. Revert this file change.

Rutu12Badwaik replied:
ok from next time will take care of this.

akashdktyagi commented on Batch8CucumberSeleniumAPI/src/test/resources/features/ui/Search.feature:
```diff
@@ -23,4 +23,14 @@ Feature: Search functionality
+		And I click on first product
+		And I clicked on Add to Cart Button
+		Then I am able to see the cart value gets updated
+		
```
Better way of writing this:

		Scenario: Validate the cart value in the header section is updated after adding product to the cart
		Given I have browser opened and url is navigated
		And I search for product as "Samsung" 
		And I click on any product in the Search Result ( Reuse this step.)
		When I clicked on Add to Cart Button on Product Description Page
		Then I am able to see the cart value gets updated in the header section

Rutu12Badwaik replied:
all changes is done

akashdktyagi commented on Batch8CucumberSeleniumAPI/src/test/resources/features/ui/Search.feature:
```diff
@@ -23,4 +23,14 @@ Feature: Search functionality
+	  
+		@TC0038
+		Scenario: Check The Updated Cart Value After Adding Products
+		Given I have browser opened and url is navigated
```
Move this test case to Add to Cart Feature

Rutu12Badwaik replied:
moved all the changes in add to cart feature

Rutu12Badwaik committed:
Merge pull request #28 from akashdktyagi/TC-0036-addToCart-Add-product-To-cart

Tc 0036 add to cart add product to cart

Rutu12Badwaik committed:
I take a pull from poanam`s branch for Add feature 
Then I make a changes in feature file which your are commited me
And I create a methods according to these steps in feature files

akashdktyagi commented:
@Rutu12Badwaik please resolve conflicts.

Rutu12Badwaik committed:
Merge branch 'develop' into R-TC0038-CartValueUpdateValidation
